src/lib.rs

```diff
@@ -88,6 +88,8 @@ pub struct BuildOutput {
     pub tool_config: tool_configuration::Configuration,
     /// Recipe path.
     pub recipe_path: PathBuf,
+    /// Resolved dependencies which are printed when render_only is used
+    pub resolved_dependencies: Vec<metadata::Output>,
```

Comment:
We are going to remove this struct pretty soon so it would be better to print this out before we return from the function I'd say.

Reply:
Unfortunately we can't print them before we return because there is a case when multiple recipes are passed
https://github.com/prefix-dev/rattler-build/pull/730/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR82

so it would be a multiple lists printed which is again un-parseable by default.

Reply:
There were some changes in master, can you resolve conflicts? I think they will make things more clear 🐻

Reply:
I've updated - let me know wdyt